### PR TITLE
Bump version to 1.0.4

### PR DESCRIPTION
**Pull request by natemcmaster**

### CODE REVIEW

File: src/Microsoft.AspNet.Identity.AspNetCoreCompat/project.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.1.3",
```
Comment (JunTaoLuo) [inline src/Microsoft.AspNet.Identity.AspNetCoreCompat/project.json]: One thing to check here is whether we want to ship this "pre-release" package. I remember I keep getting different answers when I ask this question.

Comment (JunTaoLuo) [reply src/Microsoft.AspNet.Identity.AspNetCoreCompat/project.json]: I think the last time I checked, we decided to cascade the versions but not push any new packages built to nuget, let's verify that that's still the expectation.

Comment (natemcmaster) [reply src/Microsoft.AspNet.Identity.AspNetCoreCompat/project.json]: Right. For now Microsoft.AspNet.Identity.AspNetCoreCompat is marked as "noship" in the patch system.

Comment (JunTaoLuo) [reply src/Microsoft.AspNet.Identity.AspNetCoreCompat/project.json]: Which reminds me, let's check what's in the patch packages.csv, we had issues with it in the past.

**Pull request merged**